internal/cmd/auth/login/login.go

```diff
@@ -23,6 +23,14 @@ func NewCmd(p *print.Printer) *cobra.Command {
 				"$ stackit auth login"),
 		),
 		RunE: func(cmd *cobra.Command, args []string) error {
+			p.Warn(fmt.Sprintf("%s\n%s\n%s\n%s\n%s\n\n",
+				"Starting on July 9 2024, the new STACKIT Identity Provider (IDP) will be available.",
+				"On this date, we will release a new version of the STACKIT CLI that will use the new IDP for user authentication.",
+				"This also means that the user authentication on STACKIT CLI versions released before July 9 2024 is no longer guaranteed to work for all services.",
+				"Please make sure to update your STACKIT CLI to the latest version after July 9 2024 to ensure that you can continue to use all STACKIT services.",
+				"You can find more information regarding the new IDP at https://docs.stackit.cloud/stackit/en/release-notes-23101442.html#ReleaseNotes-2024-06-21-identity-provider.",
+			))
```

Comment:
```suggestion
				"You can find more information regarding the new IDP at https://docs.stackit.cloud/stackit/en/release-notes-23101442.html#ReleaseNotes-2024-06-21-identity-provider",
			))
```

Reply:
I think the period might confuse the URL

Reply:
Works well in my terminal but good point